gcp/README.md

```diff
@@ -198,6 +198,23 @@ Examples of when you might want to do this:
 1. Prepend `USER=gitlab-runner` to all `rake` commands.
    * Or, to add an additional dev cluster: `USER=mrtyler-experiment1`
 
+### I need to work with my tfstate bucket directly
+
+For example: a developer deleted their tfstate bucket in GCS and re-created it with the wrong permissions. You need to back up the contents of the bucket so Terraform can destroy it and re-create it with the correct permissions.
+
+1. Entities in the tfstate bucket may use one of two encryption types:
+   * Google-managed - for initial bootstrapping, storage of tfstate for the secrets themselves
+   * Customer-supplied - managed with our [secrets code](https://github.com/gpii-ops/gpii-infra/blob/master/shared/rakefiles/secrets.rb)
```

Comment:
Missing "Customer managed"  - we use that for example for exported logs. (yes, we use all 3 😢types)

Reply:
Do we use customer-managed in the tfstate bucket? If so, where? If not, why would we talk about it in this section?

Reply:
ok, fair enough if it's limited to the `tfstate` bucket", I don't think we use Customer managed there.